spec/features/contextual_navigation_spec.rb

```diff
@@ -64,6 +64,40 @@
     and_the_parent_based_breadcrumbs
   end
 
+  scenario "There's a secondary step by step list and no primary step by step list" do
+    given_theres_a_page_with_a_secondary_step_by_step
+    and_i_visit_that_page
+    then_i_see_the_step_by_step
+    and_the_step_by_step_header
+  end
+
+  scenario "There's between 2-5 secondary step by step lists and no primary step by step list" do
+    given_there_are_two_secondary_step_by_step_lists
+    and_i_visit_that_page
+    then_i_just_see_the_step_by_step_related_links
+  end
+
+  scenario "There's 6 or more secondary step by step lists and no primary step by step list" do
+    given_there_are_six_secondary_step_by_step_lists
+    and_i_visit_that_page
+    then_theres_no_step_by_step_at_all
+    and_no_step_by_step_header
+  end
+
+  scenario "There's 3 secondary step by steps and 2 primary step by step lists" do
+    given_there_are_three_secondary_step_by_steps_and_two_primary_to_step_navs
+    and_i_visit_that_page
+    then_i_just_see_the_step_by_step_related_links_with_just_two_links
```

Comment:
Could this have another step confirming that we don't see the secondary step by step? Or be changed to confirm that it's definitely only the primary step by steps that in the related links?

Reply:
Done.